web/docs/Frontend Starters/Next.js/Next.js + WordPress/setting-environment-variables.md

```diff
@@ -11,10 +11,11 @@ In order to fetch data from the WordPress instance, Next.js needs to know the
 endpoint at build time. For local development, the starter kit uses
 [dotenv](https://www.npmjs.com/package/dotenv).
 
-When you clone your decoupled frontend repo, create a `.env.development.local`
-file. In this file, update the `WPGRAPHQL_URL` and `IMAGE_DOMAIN` with your
-WordPress GraphQL endpoint, and the `IMAGE_DOMAIN`. If the `WPGRAPHQL_URL` and
-`IMAGE_DOMAIN` are the same, you can omit setting the `IMAGE_DOMAIN`.
+When you generate your decoupled frontend repo, create a
+`.env.development.local` file if one does not already exist. In this file,
```

Comment:
@CobyPear I had chose this wording since as it is now we still are advertising `git clone` as an (not preferred) option so in a case where a user goes that route they would still need to create `.env.development.local`. I am open to changing this to assume they used a generator if we don't think that assumption will lead to confusion.

> Maybe we could add in a note that the WPGRAPHQL_URL/BACKEND_URL can be set as a argument when running the CLI?

I think at this point in the docs the user would have likely already generated a project, maybe that would be more suited in the CLI docs page? Could see and argument to have a note in both places as well.